src/main/java/frc/robot/Robot.java

```diff
@@ -16,14 +22,25 @@ public class Robot {
 
 	public static final RobotType ROBOT_TYPE = RobotType.determineRobotType();
 
+	private final Flywheel flywheel;
+
 	public Robot() {
+		FlywheelComponents topFlywheelComponents = FlywheelFactory
+			.create(FlywheelConstants.LOG_PATH + "TopMotor", RealFlywheelConstants.isTopMotorInverted, IDs.CANSparkMaxIDs.TOP_FLYWHEEL);
+		FlywheelComponents bottomFlywheelComponents = FlywheelFactory
+			.create(FlywheelConstants.LOG_PATH + "BottomMotor", !RealFlywheelConstants.isTopMotorInverted, IDs.CANSparkMaxIDs.BOTTOM_FLYWHEEL);
```

Comment:
Better to make another constant named isBottomMotorInverted = !isTopMotorInverted